docs/api/commands/run.md

```diff
@@ -332,23 +332,20 @@ By default, the container will be able to `read`, `write` and `mknod` these devi
 This can be overridden using a third `:rwm` set of options to each `--device`
 flag:
 
+    $ docker run --device=/dev/sda:/dev/xvdc --rm -it ubuntu fdisk  /dev/xvdc
```

Comment:
This section from lines 335-345 may be correct, but I'm not sure I understand or follow when reading it, maybe there is a duplicate line?

Reply:
The formatting is a little funky here (also line 321 is 3 spaces instead of 4 so doesn't render as a code block.)

I _think_ line 343 was meant to be:

```
$ docker run --device=/dev/sda:/dev/xvdc:w --rm -it ubuntu fdisk /dev/xvdc
```

Showing the `:w` write permission to the device as an example.